script/build

```diff
@@ -221,13 +216,23 @@ def parse() -> argparse.Namespace:
         help="Compile project with full optimisations",
     )
 
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        default=False,
+        help="Run script outside of virtualenv",
+    )
     return parser.parse_args()
 
 
 def main() -> None:
     should_setup = not (SOURCE_ROOT / "_skbuild").is_dir()
     args = parse()
 
+    if not args.force and sys.prefix == sys.base_prefix:
+        sys.exit("This script must be run from a virtualenv")
```

Comment:
Add a message where it's something akin to `This script must be run from a virtualenv, or ignore this check with --force`